packages/story-editor/src/components/publishModal/content/storyPreview.js

```diff
@@ -295,7 +292,7 @@ const StoryPreview = () => {
                     data-testid="story_preview_logo"
                   />
                 )}
-                {(hasUploadMediaAction || enablePosterHotlinking) && (
+                {hasUploadMediaAction && (
```

Comment:
Same here, with poster hotlinking always allowed we can remove this entire condition here, no?

Reply:
Not sure if you discussed this somewhere else but since now `enablePosterHotlinking` is always `true`, wouldn't we always display this and can remove `hasUploadMediaAction` check as well?